app/src/main/java/com/cookpad/android/dokumiTestAndroid/MainActivity.java

```diff
@@ -36,4 +46,30 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
         return super.onOptionsItemSelected(item);
     }
+
+    private void nullDereference() {
+        Object foo = getValueRandomly();
+        Log.e(TAG, foo.toString());
```

Comment:
**Static Analysis:** object foo last assigned on line 51 could be null and is dereferenced at line 52